gradle/publish-libs.gradle

```diff
@@ -1,35 +0,0 @@
-apply plugin: 'com.jfrog.bintray'
```

Comment:
With old publish file, immediate Gradle error that says publishing does not support multiple projects. I'd seen the error before but in the interests of progress I just ripped the http-remoting Gradle file. 

We ought to do a cleanup pass on our open source project Gradle files -- they're all sight my drifted versions of each other due to evolution...would like to get back to a standard set.